stravalib/model.py

```diff
@@ -533,9 +800,17 @@ class ActivityLap(
 
 
 class Map(PolylineMap):
+    """Pass through object. Inherits from PolyLineMap"""
+
     pass
 
 
+# if TYPE_CHECKING:
+#     from strava_model import Split
+
+
+# TODO - Split is already defined in strava_model - Name "Split" already
```

Comment:
This particularly inheritance is really confusing to me. Split i the class name but it inherits Split as well. any help here is appreciated.